provider/maas/maas_test.go

```diff
@@ -35,6 +35,19 @@ import (
 
 const maas2VersionResponse = `{"version": "unknown", "subversion": "", "capabilities": ["networks-management", "static-ipaddresses", "ipv6-deployment-ubuntu", "devices-management", "storage-deployment-ubuntu", "network-deployment-ubuntu"]}`
 
+const maas2DomainsResponse = `
+[
+    {
+        "authoritative": "true",
+        "resource_uri": "/MAAS/api/2.0/domains/0/",
+        "name": "maas",
```

Comment:
Do we want to return more than one domain in testing?  One that is what we're looking for and one that isn't. And I just noticed this is a back port....